jumpscale/sals/marketplace/apps_chatflow.py

```diff
@@ -386,15 +386,13 @@ def _get_domain(self):
             )
 
     def _config_logs(self):
-        self.solution_log_config = j.core.config.get("LOGGING_SINK", {})
+        self.solution_log_config = self.nginx_log_config = self.trc_log_config = j.core.config.get("LOGGING_SINK", {})
         if self.solution_log_config:
-            self.solution_log_config["channel_name"] = self.solution_name
-        self.nginx_log_config = j.core.config.get("LOGGING_SINK", {})
-        if self.nginx_log_config:
-            self.nginx_log_config["channel_name"] = self.solution_name + "-nginx"
-        self.trc_log_config = j.core.config.get("LOGGING_SINK", {})
-        if self.trc_log_config:
-            self.trc_log_config["channel_name"] = self.solution_name + "-trc"
+            self.solution_log_config["channel_name"] = f"{self.threebot_name}-{self.SOLUTION_TYPE}-{self.solution_name}"
+            self.nginx_log_config[
+                "channel_name"
+            ] = f"{self.threebot_name}-{self.SOLUTION_TYPE}-{self.solution_name}-nginx"
+            self.trc_log_config["channel_name"] = f"{self.threebot_name}-{self.SOLUTION_TYPE}-{self.solution_name}-trc"
```

Comment:
maybe we can do same checks as above ?

Reply:
it is the same dict/config :)

Reply:
this one in case we didn't override in the chatflow that inherits from it, hence they are the defaults, we shouldn't allow that undersocres too, right ? :D